packages/office-ui-fabric-react/src/components/Check/Check.scss

```diff
@@ -36,12 +37,10 @@ $checkBoxHeight: 24px;
   font-size: $checkBoxHeight;
 }
 
-.background {
-  color: $ms-color-white;
-}
-
-.rootIsChecked .background {
-  color: $ms-color-themePrimary;
+.rootIsChecked {
+  &.root {
```

Comment:
can you move this section to be a child of the .root block above, for readability?